express/parsers/structure.py

```diff
@@ -1,6 +1,12 @@
 import io
 import pymatgen as mg
+from pymatgen import io as io
 from ase.io import read, write
+import openbabel
```

Comment:
This openbabel line will always fail in the github tests because openbabel is not properly available outside of butler's venv.